pkg/controllers/consistency/nodeshape.go

```diff
@@ -60,19 +60,19 @@ func (n *NodeShape) Check(ctx context.Context, node *v1.Node) ([]Issue, error) {
 	}
 	instanceType, ok := lo.Find(instanceTypes, func(it *cloudprovider.InstanceType) bool { return it.Name == node.Labels[v1.LabelInstanceTypeStable] })
 	if !ok {
-		return []Issue{Issue(fmt.Sprintf("Instance Type %q not found", node.Labels[v1.LabelInstanceTypeStable]))}, nil
+		return []Issue{Issue(fmt.Sprintf("instance Type %q not found", node.Labels[v1.LabelInstanceTypeStable]))}, nil
 	}
 	var issues []Issue
 	for resourceName, expectedQuantity := range instanceType.Capacity {
 		nodeQuantity, ok := node.Status.Capacity[resourceName]
 		if !ok && !expectedQuantity.IsZero() {
-			issues = append(issues, Issue(fmt.Sprintf("Expected resource %s not found", resourceName)))
+			issues = append(issues, Issue(fmt.Sprintf("expected resource %s not found", resourceName)))
```

Comment:
This appears as an event too. Is it correct to have no capatalization at the beginning of the sentence for events?

Reply:
This is used for both logging and metrics. If I had to pick one, it would be lower case since it's easier to stitch with other context log snippits.